common/views/components/SelectUncontrolled/README.md

```diff
@@ -1,4 +1,5 @@
 ## Purpose
-A styled select element.
 
-[Edit this on GitHub](https://github.com/wellcometrust/wellcomecollection.org/edit/master/common/views/components/Select/README.md)
+A styled select element uncontrolled with JS.
```

Comment:
nit: 'not controlled by JS' maybe sounds a bit more human-friedly?

Reply:
Went through the same thing in my head.